docs/src/pages/guides/new-notifications.md

```diff
@@ -0,0 +1,173 @@
+---
+id: new-notifications
+title: New Notifications
+---
+
+## New Notifications
+
+Out of the box, Monika support a multitude of notification channels, from chat applications such as Slack, Discourse, Microsoft Teams, Google Chats to email based platform such as Mailgun, to generic webhooks and the SMTP protocol ([click here for the full list](https://monika.hyperjump.tech/guides/notifications)). Should you have a favorite tool not already supported, we invite you to add it to Monika's growing list. Here's how.
```

Comment:
click here for the full list -> full list can be found in Notifications
favorite tool not already supported -> favorite tool we haven't supported,

Reply:
done. thanks.